spec/features/users_can_upload_actuals_spec.rb

```diff
@@ -147,7 +147,7 @@ def expect_to_see_successful_upload_summary_with(count:, total:)
       expect(page).to have_xpath("td[1]", text: "Actual Value")
       expect(page).to have_xpath("td[2]", text: "2")
       expect(page).to have_xpath("td[3]", text: "fish")
-      expect(page).to have_xpath("td[4]", text: t("importer.errors.actual.non_numeric_value"))
+      expect(page).to have_xpath("td[4]", text: "Actual and refund values must be blank or numeric")
```

Comment:
Is there a reason you've made assertions on real text values, rather than their translations? I've seen this in a few tests in this PR, which would mean we'd need to update them in both places if the translated text were to change. Not a huge deal, but just wondered if it was a testing preference?

Reply:
I think it might be a preference of @CristinaRO's (and @jkempster34's) to explicitly include the text in the test to make it more readable. I'm quite keen on DRY code, but I can see that it reads as documentation better when explicit. Open to whichever approach!

Reply:
So for me, testing against the translations:
1. Hides the actual text making the test harder to read.
2. If the translation is missing, the test will still pass (I think) because both will be nil. Kind of relates [to not using logic in tests](https://canro91.github.io/2021/10/11/DontRepeatLogicInAssertions/)
3. The downside of changing the text in both places isn't a big deal, its just a small annoyance.

But I don't have really strong feelings!

Reply:
Feels like a sea of ambivalence 😅

Reply:
I don't mind either way; I think originally I wrote the specs as a demo and didn't want to consider the translation keys and values as firmly decided, and they were left like that 😅